### PR TITLE
EDGEINV-3: Add support for ECS without DCB

### PR DESCRIPTION
**Pull request by izhunke**
Add support for ECS details: https://folio-org.atlassian.net/browse/EDGEINV-3

### CODE REVIEW

File: src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+@RequiredArgsConstructor
+@Log4j2
+public class EcsInventoryService {
+
+  private final UsersClient usersClient;
+  private final InventoryClient inventoryClient;
+  private final SearchClient searchClient;
+  private final JsonConverter jsonConverter;
+  private final FolioExecutionContext folioExecutionContext;
+  public final String FACET = "holdings.tenantId";
```
Comment (VitaliiZahoruiko) [inline src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java]: (minor) could be static?

File: src/main/java/org/folio/edge/inventory/service/EcsLocationsService.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import org.folio.edge.inventory.client.SearchClient;
+import org.folio.edge.inventory.util.JsonConverter;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class EcsLocationsService {
+  private final SearchClient searchClient;
+  private final JsonConverter jsonConverter;
+  public static final String LOCATIONS = "locations";
```
Comment (VitaliiZahoruiko) [inline src/main/java/org/folio/edge/inventory/service/EcsLocationsService.java:16]: public static variable should be declared before non-static (on top)

File: src/main/java/org/folio/edge/inventory/util/JsonNodeUtil.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package org.folio.edge.inventory.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
```
Comment (VitaliiZahoruiko) [inline src/main/java/org/folio/edge/inventory/util/JsonNodeUtil.java:6]: @UtilityClass can be added

**Review by VitaliiZahoruiko — APPROVED** [review]

File: src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java (hunk 1 above)
Comment (izhunke) [reply src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java]: fixed

File: src/main/java/org/folio/edge/inventory/service/EcsLocationsService.java (hunk 2 above)
Comment (izhunke) [reply src/main/java/org/folio/edge/inventory/service/EcsLocationsService.java:16]: fixed

File: src/main/java/org/folio/edge/inventory/util/JsonNodeUtil.java (hunk 3 above)
Comment (izhunke) [reply src/main/java/org/folio/edge/inventory/util/JsonNodeUtil.java:6]: fixed

File: src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+          if (response.getFacets().getHoldingsTenantId().getTotalRecords() == 0) {
+            return new InstanceTenants(instanceId, Collections.EMPTY_LIST);
+          }
+          var tenantIds = response.getFacets().getHoldingsTenantId().getValues()
+              .stream().map(value -> value.getId()).toList();
+          return new InstanceTenants(instanceId, tenantIds);
+        })).toList();
+  }
+
+  public boolean isCentralTenant(String tenantId) {
```
Comment (illia-borysenko) [inline src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java]: Please move public method to the top of the class

**Review by illia-borysenko — APPROVED** [review]

**Comment by illia-borysenko** [comment]
@izhunke Can you please recreate commits under Folio github account, there is a problem with license, seems you created commits under Ebsco account

File: src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java (hunk 4 above)
Comment (izhunke) [reply src/main/java/org/folio/edge/inventory/service/EcsInventoryService.java]: moved

**Pull request merged**